flink-connector-kafka-e2e-tests/flink-end-to-end-tests-common-kafka/src/test/java/org/apache/flink/tests/util/kafka/SQLClientKafkaITCase.java

```diff
@@ -82,15 +86,27 @@ private static Configuration getConfiguration() {
         return flinkConfig;
     }
 
-    @Rule
```

Comment:
Refactored to remove the `flink-end-to-end-tests-common` dependency

Reply:
I actually propose to get rid of this test. Reason being is that SQLClientSchemaRegistryITCase tests SQL Kafka table source and sink already.

This test is just testing E2E the json format -> avro format -> csv format via file sink. However, now that we are using containerized Flink, the file it writes to is now within the container and it is difficult to obtain the file from the taskmanager containers to assert on the file output contents.

We could easily fix this test if we can use LocalStandaloneFlinkResourceFactory (but it is in flink-end-to-end-test-common). 

Plus I had to create an extraneous module to get the jar of a custom sql function (see flink-sql-toolbox module). 

Alternatively, I can totally rewrite and simplify the test. (only testing json format -> avro format and asserting Kafka output contents, but this is what SQLClientSchemaRegistryITCase is essentially doing).

@zentol @MartijnVisser WDYT?

Reply:
(there was previous feedback to try un-ignoring this test)

Reply:
@MartijnVisser are we just waiting for Chesnay to come back from vacation to continue this conversation? I had wanted to get this connector ready before the 1.16.1 release. This is the only test that is failing in CI

Reply:
@zentol @MartijnVisser hope you enjoyed your break! Do you have to time to revisit this discussion^?

It's been a while so just reminding--this is a e2e test that was previously @Ignore'ed and you left a comment to try and re-enable it since the commented JIRA ticket that was blocking this test was fixed. It's possible to support this with some improvements of the Flink testcontainer framework (I can go into the details of what is required). But I think the E2E test is redundant as per my earlier comment on Dec 15.

Reply:
I'd be fine with dropping this test.